src/main/java/tombenpotter/sanguimancy/Sanguimancy.java

```diff
@@ -66,40 +67,30 @@ public void preInit(FMLPreInitializationEvent event) {
         RecipesRegistry.registerShapedRecipes();
         RecipesRegistry.registerOrbRecipes();
         EntitiesRegistry.registerEntities();
-        PotionsRegistry.potionPreInit();
+        PotionsRegistry.registerPotions();
         if (ModList.computercraft.isLoaded()) PeripheralProvider.register();
-        RandomUtils.createSNDimension();
+//        RandomUtils.createSNDimension();
     }
 
     @Mod.EventHandler
     public void load(FMLInitializationEvent event) {
         proxy.load();
-        RitualRegistry.registerRituals();
+        RitualsRegistry.registerRituals();
         RecipesRegistry.registerAltarRecipes();
-        RecipesRegistry.registerAlchemyRecipes();
         RecipesRegistry.registerBindingRecipes();
-        PotionsRegistry.registerPotions();
         FMLCommonHandler.instance().bus().register(new EventHandler());
         MinecraftForge.EVENT_BUS.register(new EventHandler());
-        FMLCommonHandler.instance().bus().register(TeleportingQueue.getInstance());
         PacketHandler.registerPackets();
-        if (Loader.isModLoaded("Waila")) {
-            WailaCompatRegistry.register();
-        }
+//        if (Loader.isModLoaded("Waila")) {
+//            WailaCompatRegistry.register();
+//        }
         NetworkRegistry.INSTANCE.registerGuiHandler(this, proxy);
     }
 
-    @Mod.EventHandler
-    public void imcCallback(FMLInterModComms.IMCEvent event) {
-        for (final FMLInterModComms.IMCMessage imcMessage : event.getMessages()) {
-            MessageRegistry.registerMessage(imcMessage.key, imcMessage);
-        }
-    }
-
     @Mod.EventHandler
     public void postInit(FMLPostInitializationEvent event) {
         RecipesRegistry.registerCustomModRecipes();
-        SanguimancyGuide.registerGuide();
```

Comment:
GuideAPI has changed a LOT since 1.10, I'm not sure how to register the book now since its no longer `static`